test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -523,6 +523,10 @@ def test_make_text_prop(self) -> None:
                 'text <text text="text" > text',
                 'text &lt;text text="text" &gt; text',
             ],
+            [
+                "text &amp; text",
+                "text &amp;amp; text",
+            ],
```

Comment:
these could/should be tuples, as it's always a pair and doesn't need the variable size of the list data type. Also, more stylistically: I'd prefer having them on one line, so each test case takes up only one line, not four.

Reply:
I turned them into tuples, but I prefer them on 4 lines, so that input and output are vertically aligned.

Reply:
Another thing you could consider: I've been pushing for less of those parametrized tests for a long time, and I was under the impression that lately, those have become a bit less (which to me is far more clear, both when reading tests, and when reading test reports), so you could re-write this to a non-parametrized variant of the same tests. But of course you don't have to

Reply:
It was indeed you good influence on me that led me away from writing new parametrized tests. If there is opportunity, I de-parametrize old tests. But the entire excel2xml code has very much room for improvement, and I could easily get lost if I start refactoring too much. So I won't do it in this PR.

Reply:
sure!